spring-data-jpa-batch-dml/src/main/java/com/baeldung/jpa/controller/CustomerController.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.jpa.controller;
```

Comment:
Why do we have two `CustomerController`s now? (As well as two of several other things) The duplication seems unnecessary.